daemon/labels.go

```diff
@@ -142,20 +142,20 @@ func (d *Daemon) CreateOrUpdateIdentity(lbls labels.Labels, epid string) (*polic
 	return identity, isNew, nil
 }
 
-func (d *Daemon) updateContainerIdentity(contID, oldLabelsHash string, opLabels *labels.OpLabels) (*policy.Identity, string, error) {
+func (d *Daemon) updateEndpointIdentity(epID, oldLabelsHash string, opLabels *labels.OpLabels) (*policy.Identity, string, error) {
 	lbls := opLabels.Enabled()
-	log.Debugf("Container %s is resolving identity for labels %+v", contID, lbls)
+	log.Debugf("Endpoint %s is resolving identity for labels %+v", epID, lbls)
 
 	newLabelsHash := lbls.SHA256Sum()
-	identity, _, err := d.CreateOrUpdateIdentity(lbls, contID)
```

Comment:
this won't work for mesos though. We create the endpoint in CNI plugin before we receive a create-container event. OK, I might have misunderstood @tgraf's comment. Leaving it "as is" should be read as is in my PR or as it was before?